Makefile.core.mk

```diff
@@ -668,17 +644,17 @@ e2e_files = istio-auth-non-mcp.yaml \
 			istio-one-namespace-auth.yaml \
 			istio-one-namespace-trust-domain.yaml \
 			istio-multicluster.yaml \
-			istio-multicluster-split-horizon.yaml \
-
-FILES_TO_CLEAN+=install/consul/istio.yaml \
-                install/kubernetes/istio-auth.yaml \
-                install/kubernetes/istio-citadel-plugin-certs.yaml \
-                install/kubernetes/istio-citadel-with-health-check.yaml \
-                install/kubernetes/istio-one-namespace-auth.yaml \
-                install/kubernetes/istio-one-namespace-trust-domain.yaml \
-                install/kubernetes/istio-one-namespace.yaml \
-                install/kubernetes/istio.yaml \
-                samples/bookinfo/platform/consul/bookinfo.sidecars.yaml \
+			istio-multicluster-split-horizon.yaml
+
+FILES_TO_CLEAN+=$(ISTIO_OUT)/install/consul/istio.yaml \
```

Comment:
nit:we already clean the whole ISTIO_OUT

Reply:
I removed $(ISTIO_OUT) prefix to all of these dirs, so they continue to need cleaning.